superbench/benchmarks/model_benchmarks/pytorch_bert.py

```diff
@@ -23,7 +28,32 @@ def __init__(self, config, num_classes):
             num_classes (int): The number of objects for classification.
         """
         super().__init__()
-        self._bert = BertModel(config)
+        self._enable_fp8 = False
```

Comment:
maybe this can be moved into model_base I think? it should be general for all models to support fp8 precision？